### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -55,29 +59,17 @@ runs:
...
+            build \
+            --arch=x86_64 \
+            --sbom=false \
+            wolfi-act.apko.config.yaml \
+            wolfi-act:latest \
+            wolfi-act.tar "${debug_args}"
+
         echo "done."
 
         printf "[🐙] Loading ephemeral container image into Docker... "
```
Comment (jdolitsky) [inline action.yml:75]: one more here!

Comment (cpanato) [reply action.yml:75]: gotcha! thanks!

**Review by jdolitsky — COMMENTED** [review]
hm, looks like lots of debug lines still in the non-debug run?

**Comment by mattmoor** [comment]
Y'all still interested in this?

**Comment by cpanato** [comment]
the `[🐙] Enabling debug logging.` only shows up in the debug job the other seems ok

**Comment by cpanato** [comment]
@mattmoor @jdolitsky PTAL

**Review by jdolitsky — APPROVED** [review]
nice!

**Pull request merged**